core/src/main/java/jenkins/widgets/JenkinsWidgetFactory.java

```diff
@@ -35,6 +35,7 @@
 
 /**
  * Add widgets annotated with @Extension, or added manually to Jenkins via <code>Jenkins.get().getWidgets().add(...)</code>
+ * @since TODO
```

Comment:
```suggestion
```

If it is not part of the API surface, then it does not matter when it was added.